docs/project/set-up-dev-env.md

```diff
@@ -255,25 +255,25 @@ example, you'll edit the help for the `attach` subcommand.
     Your location should be different because, at least, your username is
     different.
 
-3. Open the `api/client/attach.go` file.
+3. Open the `api/client/container/attach.go` file.
 
 4. Edit the command's help message.
 
     For example, you can edit this line:
 
-    ```go
-    noStdin := cmd.Bool([]string{"-no-stdin"}, false, "Do not attach STDIN")
-    ```
+	```go
+	flags.BoolVar(&opts.noStdin, "no-stdin", false, "Do not attach STDIN")
+	```
 
     And change it to this:
 
     ```go
-    noStdin := cmd.Bool([]string{"-no-stdin"}, false, "Do not attach STDIN (standard in)")
+	flags.BoolVar(&opts.noStdin, "no-stdin", false, "Do not attach STDIN (standard in)")
     ```
 
-5. Save and close the `api/client/attach.go` file.
+5. Save and close the `api/client/container/attach.go` file.
 
-6. Go to your running development container.
```

Comment:
The original instructions actually did not work for me. Am I missing something? It looks like the code changes were not automatically included in the running development container. 

After exiting that container and running `make shell` I was able to see my code changes.